cmd/cerbosctl/decisions/decisions.go

```diff
@@ -11,33 +11,27 @@ import (
 	"strings"
 	"time"
 
+	"github.com/alecthomas/kong"
+	client2 "github.com/cerbos/cerbos/cmd/cerbosctl/internal/client"
```

Comment:
Let's use a more descriptive alias like `cmdclient` instead of `client2`. There are a few more like this in the files below (`policy2`, `client2`) as well